app/src/main/java/com/example/ana/exampleapp/TestActivity.java

```diff
@@ -41,6 +46,8 @@
     private FeedTestDbHelper mDbHelper;
     private final static String[] projection = {
             FeedTestContract.FeedEntry.COLUMN_NAME_TIMESTAMP,
+            FeedTestContract.FeedEntry.COLUMN_LATITUDE,
+            FeedTestContract.FeedEntry.COLUMN_LONGITUDE,
```

Comment:
You do not need the longitude and latitude in the projection, as it is used when the user wants to change his answers to fill in the test with his previous answers. In addition, you add them before the rest of the columns when doing `c.getInt(i)` we won't access to the correct column